concordium-node/src/bin/cli.rs

```diff
@@ -608,6 +608,7 @@ async fn import_missing_blocks(
         genesis_hash
     );
 
+    let mut mayskip = true;
```

Comment:
```suggestion
    // The `last_finalized_block_height` is relative for the starting genesis index.
    // If we are catching up beyond a protocol update we need to start from 0
    // at that genesis index. Instead of observing whether a protocol update has
    // occurred we instead go with the simpler, but equally good solution,
    // of only skipping some chunks at the beginning of the import.
    // This flag indicates whether we have stopped skipping at the beginning of the
    // out of band catchup.
    let mut mayskip = true;
```

Reply:
Formatting is probably off.